File: JS PS/q20.js
// 20. Find and print the difference between the maximum and 
// minimum elements in arr = [80, 30, 70, 50, 20].

let arr = [80, 30, 70, 50, 20]
console.log(Math.max(...arr)- Math.min(...arr))

//

let max=arr[0], min=arr[0];
for(i=0;i<arr.length;i++){
    if(max<arr[i]){
        max=arr[i]
    }
    if(min>arr[i]){
        min=arr[i]
    }
}
console.log(max-min)

File: Clang PS/Q20.c
#include <stdio.h>
int main() {
   int arr[]={80, 30, 70, 50, 20};
   int min=arr[0];
   int max=arr[0];
   int diff;
   int i;
   int len=sizeof(arr)/sizeof(arr[0]);
   for(int i=0;i<len;i++){
       if(min>arr[i]){
         min=arr[i];
       }
       if(max<arr[i]){
        max=arr[i];
       } 
       
   } 
    diff=max-min;    
    printf("%d",diff);
   
    return 0;
}